dna-diffusion/requirements.yml

```diff
@@ -0,0 +1,29 @@
+name: dnadiffusion
+channels:
+  - conda-forge
+  - bioconda
+  - pytorch
+  - nvidia
+dependencies:
+  - pytorch
+  - pytorch-cuda=11.6
+  - torchvision
+  - torchaudio
+  - accelerate
+  - jupyterlab
+  - matplotlib
+  - pandas
+  - seaborn
+  - einops
+  - wandb
+  - genomepy
+  - pip
+  - pip:
```

Comment:
perhaps search https://conda-forge.org/feedstock-outputs/ to try to minimize pip installation to only packages that are not on conda-forge